docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: december-2023
+append_help_link: true
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: December 2023
+tags:
+  - Release notes
+---
+
+# December 2023 release notes
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the repository so that it
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE Extensions. These changes
+  persist between restarts.
+* Added authentication provider information for each user under **Settings** >
+  **Access** > **Members** to help distinguish between users who log in with
+  multiple methods.
+* The metrics collected now include more granular information to help
+differentiate scans using different engine capabilities, such as intraprocedural
+scans without secrets validation versus intraprocedural scans *with* secrets
+validation.
+* **CLI tool**: Added new `semgrep test` subcommand, which is an alias for `semgrep scan
+--test`. **Note**: When running Semgrep on a test directory, use `semgrep
+scan test` to avoid confusion with the new `semgrep test` subcommand.
+
+### Changed
+
+* Improved the speed with which Semgrep Cloud Platform's **Dashboard** and **Findings**
```

Comment:
This is still being rolled out globally. I wonder if we should wait to put this into Jan's notes when everyone will have it?

Reply:
Will make a note to include in Jan